website/src/css/customTheme.scss

```diff
@@ -43,7 +43,6 @@
   --ifm-table-border-color: var(--ifm-toc-border-color);
   --ifm-table-cell-padding: 10px;
   --ifm-table-stripe-background: rgba(0, 0, 0, 0.02);
-  --docusaurus-blog-social-icon-size: 16px;
```

Comment:
This was not used, and after introducing cascade layers, it became used and produced a visual diff that my tool caught.

The effective icon size on the RN website in prod is 17px, not 16px.